docs/docs/conceptual/gatsby-for-ecommerce.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using Gatsby For E-commerce
+---
+
+Businesses selling products online typically need a variety of software to support their experience. At a minimum, their website needs product pages, product catalog navigation, a shopping cart, and checkout. 
+
+Most have additional functionality like customer account creation, discount codes, customer reviews, tax calculation, user tracking via analytics, and content personalization.
+
+The website functions because these front-end capabilities integrate with a wide swath of software on the backend, such as inventory management, order fulfillment, business analytics, and customer engagement via email. 
+
+### Choosing a main e-commerce platform
+
+Most businesses choose a central e-commerce platform as their source of truth for these functionality. Some businesses choose to run entirely on these platforms, which can be quicker to get started with but lock you in to their choices for website creation (eg Liquid Templates for Shopify). 
+
+If you're using or considering Gatsby, your organization is likely optimizing for specific properties of your website (such as performance, development environment, design, user experience) and are therefore considering a "JAMstack", "decoupled" or "content mesh" architecture. 
```

Comment:
Security I think is another key reason to enumerate

Reply:
Good point!